tests/test_cfp.py

```diff
@@ -275,8 +275,8 @@ def test_ignores_new_tags(self):
         assert Tag.objects.count() == 0
 
     def test_ignores_new_tags_keeping_predefined_ones(self):
-        ConferenceTag.objects.create(name='django')
-        ConferenceTag.objects.create(name='love')
+        ConferenceTag.objects.create(name=u'django')
+        ConferenceTag.objects.create(name=u'love')
```

Comment:
not sure if necessary

Reply:
I think this was generating a warning, hence the change.